test/test_torch.py

```diff
@@ -6166,10 +6166,11 @@ def test_equal(self):
         self.assertEqual(neg_0.stride(), neg_1.stride())
         self.assertEqual(neg_0.size(), neg_1.size())
         self.assertFalse(torch.equal(neg_0, neg_1))
-        # Disable the following check due to the inductor failure
+        # FIXME: Disable the following check due to the inductor failure
         # See https://github.com/pytorch/pytorch/issues/100340 and
         # https://github.com/pytorch/pytorch/issues/98175
-        # self.assertTrue(torch.equal(neg_0, neg_1._neg_view()))
+        if not TEST_WITH_TORCHINDUCTOR:
```

Comment:
I think the comment suggests that you could decorate the test with `@skipIfTorchInductor("REASON)`, for example:

```
@skipIfTorchInductor("See https://github.com/pytorch/pytorch/issues/98175")
def test_equal(self):
    ...
    # Run test as normal
    self.assertTrue(torch.equal(neg_0, neg_1._neg_view()))
```

Reply:
We still want to most part of tests, right?

Reply:
And the file already has the of TEST_WITH_TORCHINDUCTOR